irods/models.py

```diff
@@ -22,22 +22,19 @@ class Model(six.with_metaclass(ModelBase, object)):
 class RuleExec(Model):
     id = Column(Integer, 'RULE_EXEC_ID', 1000)
     name = Column(String, 'RULE_EXEC_NAME', 1001)
+    rei_file_path = Column(String,'RULE_EXEC_REI_FILE_PATH', 1002)
     user_name = Column(Integer, 'RULE_EXEC_USER_NAME', 1003)
+    exec_time = Column(DateTime,'RULE_EXEC_TIME',    1005)
```

Comment:
consider... just making this 'time'?

Reply:
If so, should  `last_exe_time` also be changed to `last`?

Reply:
i don't think so...

- 'time' is the next scheduled time
- 'last_exe_time' is the last time it fired

could have an overload/convenience accessor where:
- 'next' -> 'time'
- 'previous' -> 'last_exe_time'

but then we're making up new words...